src/event/event_dispatcher.cpp

```
@@ -4,26 +4,27 @@
 #include <tuple>
 #include <utility>
 
-EventDispatcher::EventDispatcher() {}
+#include <volasim/event/event.h>
+#include <volasim/event/event_listener.h>
 
 EventDispatcher::~EventDispatcher() {
```

Clang (14.0.6)

[error] use '= default' to define a trivial destructor

(modernize-use-equals-default,-warnings-as-errors)